src/node/internal/streams_util.js

```diff
@@ -116,6 +116,25 @@ export function isNodeStream(obj) {
   );
 }
 
+export function isReadableStream(obj) {
```

Comment:
We have the same implementation in src/cloudflare/internal/images-api.ts line 308. Can we remove the duplicated one?

Reply:
Hmm, they have different implementations though:

cloudflare/internal/images-api.ts
```ts
function isReadableStream(obj: unknown): obj is ReadableStream {
  return !!(
    obj &&
    typeof obj === 'object' &&
    'getReader' in obj &&
    typeof obj.getReader === 'function'
  );
}
```

streams_util.js
```js
export function isReadableStream(obj) {
  return !!(
    obj &&
    !isNodeStream(obj) &&
    typeof obj.pipeThrough === 'function' &&
    typeof obj.getReader === 'function' &&
    typeof obj.cancel === 'function'
  );
}
```

do you feel this is a safe change to unify the Cloudflare implementation with?